Kitodo/src/test/java/org/kitodo/config/ConfigProjectTest.java

```diff
@@ -67,7 +67,7 @@ public void shouldGetTifDefinition() {
     @Test
     public void shouldGetTitleDefinition() {
         String titleDefinition = configProject.getTitleDefinition();
-        String expected = "ATS+TSL+'_'+PPN digital a-Satz";
+        String expected = "TSL_ATS+'_'+CatalogIDDigital";
```

Comment:
I assume you changed the code to english in generation? Thats why the identifiers change?
May it be a problem with old processes?

Reply:
No, I didn't make a translation to english. But instead of using the labels in the title definition  and tifHeader definition I have used the metadataIds.